client/python/test/test_serializing.py

```diff
@@ -22,7 +22,9 @@ def test_duplicate(serializer_class):
 def test_check_component_fallback_serializers():
     for component_name, class_type in COMPONENT_MAPPING.items():
         # skip components that cannot be instantiated without parameters
-        if component_name in ["dataset", "interpretation"]:
+        if component_name in ["dataset", "interpretation"] + list(
+            SKIP_COMPONENTS - {"state"}
```

Comment:
Should we just add `["dataset", "interpretation"]` to SKIP_COMPONENTS and remove them from COMPONENT_MAPPING?